etc/jshintrc.server

```diff
@@ -1,8 +0,0 @@
-{
-    "esversion": 6,
-    "undef": true,
-    "node": true,
-    "mocha": true,
-    "expr": true,
```

Comment:
I didn't find an equivalent in ESLint for this one, but I think it's covered in some sense.